docs/rest_api/index.rst

```diff
@@ -0,0 +1,20 @@
+Bodhi REST API
+==============
+
+This section of the documentation describes Bodhi's REST API. You can read about the various
+sections of the API by following the links below:
+
+
+.. toctree::
+   :maxdepth: 2
+
+   builds
```

Comment:
It was actually in one file when I started working on this and it was huge so I actually broke it up on purpose.